items/not_answered.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/contexts/generic.jsonld",
+    ],
+    "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Field.jsonld",
+    "@id": "not_answered",
+    "skos:prefLabel": "Not Answered option",
+    "skos:altLabel": "not_answered",
+    "schema:description": "Not Answered option",
+    "schema:schemaVersion": "0.0.1",
+    "schema:version": "0.0.1",
+    "question": null,
+    "ui": {
+        "inputType": "checkbox"
+    },
+    "responseOptions": {
+        "type": "xsd:string",
+        "schema:name": "Not Answered",
+        "schema:value": "not_answered"
+    }
```

Comment:
this i think should be merged with the constants

Reply:
i wasn't aware that these additional response options were being defined as constants. we had talked about defining them as fields so that their input type is decided on in the schema and not hard coded in the renderer. @sanuann @akeshavan thoughts on how we move forward?